misk-hibernate/src/main/kotlin/misk/hibernate/RealTransacter.kt

```diff
@@ -166,12 +166,14 @@ internal class RealTransacter private constructor(
     override fun <T> target(shard: Shard, function: () -> T): T {
       if (config.type == DataSourceType.VITESS) {
         return useConnection { connection ->
-          val previousTarget = connection.createStatement().use { statement ->
-            val rs = statement.executeQuery("SELECT database()")
-            check(rs.next())
-            val previousTarget = rs.getString(1)
+          // This broke on master Vitess so we temporarily disable it.
+          val previousTarget = ""
+          // TODO re-enable this when it's working again: https://github.com/vitessio/vitess/pull/3962
+//          val previousTarget = connection.createStatement().use { statement ->
```

Comment:
nit: delete code or use `if (false)` rather than commenting it out. It’s very scary to uncomment commented-out code because it doesn’t participate in refactorings.

Reply:
Nice! DOne!